src/tests/functional/plugin/shared/src/snippets/mha.cpp

```diff
@@ -15,64 +15,67 @@ namespace test {
 namespace snippets {
 
 std::string MHA::getTestCaseName(testing::TestParamInfo<ov::test::snippets::MHAParams> obj) {
-    std::vector<ov::PartialShape> inputShapes;
+    std::vector<InputShape> input_shapes;
     std::vector<ov::element::Type> elem_types;
     ov::element::Type prc;
-    bool withMul;
+    bool with_mul;
     size_t thread_count;
-    std::string targetDevice;
+    std::string target_device;
     size_t num_nodes, num_subgraphs;
-    ov::AnyMap additionalConfig;
-    std::tie(inputShapes,
+    ov::AnyMap additional_config;
+    std::tie(input_shapes,
              elem_types,
              prc,
-             withMul,
+             with_mul,
              thread_count,
              num_nodes,
              num_subgraphs,
-             targetDevice,
-             additionalConfig) = obj.param;
+             target_device,
+             additional_config) = obj.param;
 
     std::ostringstream result;
-    for (size_t i = 0; i < inputShapes.size(); ++i)
-        result << "IS[" << i << "]=" << ov::test::utils::partialShape2str({inputShapes[i]}) << "_";
+    for (size_t i = 0; i < input_shapes.size(); i++)
+        result << "IS[" << i << "]=" << input_shapes[i] << "_";
     for (size_t i = 0; i < elem_types.size(); i++)
         result << "T[" << i << "]=" << elem_types[i] << "_";
-    result << "Mul=" << withMul << "_";
+    result << "Mul=" << with_mul << "_";
     result << "ThreadNum=" << thread_count << "_";
     result << "PRC=" << prc << "_";
     result << "#N=" << num_nodes << "_";
     result << "#S=" << num_subgraphs << "_";
-    result << "targetDevice=" << targetDevice << "_";
+    result << "targetDevice=" << target_device << "_";
 
-    if (!additionalConfig.empty()) {
+    if (!additional_config.empty()) {
         result << "_PluginConf";
-        for (auto& item : additionalConfig) {
+        for (auto& item : additional_config) {
             result << "_" << item.first << "=" << item.second.as<std::string>();
         }
     }
-    return result.str();
+    // todo: the following lines are needed for correct tests representation in clion ide. remove before merge
+    auto res = result.str();
+    std::replace(res.begin(), res.end(), ',', '.');
```

Comment:
Aha, now it's clear :) Let's remove this code here in matmul.cpp, and in transpose_matmul.cpp in accordance with the comment

Reply:
Removed in all 3 files, thanks